### ISSUE TITLE
Contact list not showing after sorting by subscriptionStatus

### ISSUE THREAD

**Issue by eddiedale** [putyourlightson/craft-campaign#374]
### Steps to reproduce the issue.
Go to campaign/contacts/ and click "sort after Subscription Status"
Probably dependent on the actual contact list, it may of may not be an issue, but for me I then get a "Server Error".

Refreshing the page will thereafter show an empty contact list, and a server-error notification.

The problem persists also after changing the sort after to another column.

### Log message
`PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'subscriptionStatus' in 'order clause' in /home/forge/www.site.com/vendor/yiisoft/yii2/db/Command.php:1302`

### Plugin and Craft version numbers.

```
PHP 8.1.10
MySQL 8.0.32

Craft Pro 4.4.5

Campaign (Lite) 2.5.5
Copy 2.3.0
Imager X 4.1.12
Plausible 2.0.0
Postmark 3.0.0
Redactor 3.0.4
Retcon 2.6.1
Retour 4.1.11
SEOmatic 4.0.23
Snaptcha 4.0.1
```

**Comment by bencroker** [comment]
Thanks for reporting this, I’ll work on a releasing a fix in the coming days.

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-campaign/commit/d9027147afc21133e521a0444ed76eacb9165e21 for the next release.

**Comment by bencroker** [comment]
Released in version 2.6.0.
